server/conf/i18n/messages

```diff
@@ -125,6 +125,9 @@ banner.clientInfoUpdated=Client info has been successfully updated.
 banner.clientAcctDeleted=The client account for "{0}" has been successfully deleted.
 # Message when a user successfully creates a client account.
 banner.newClientCreated=New client successfully created.
+# Banner at the top of the page when viewing an application
```

Comment:
**nitpick:** Could we add: "with the name of the client substituted in". In general, its good to explain what will be put in for the {0}s when those are included, since it may not always be clear to the translator.

Reply:
yeah for sure. added.